samples/excel/range/selected-range.xaml

```diff
@@ -3,66 +3,46 @@ gist: ''
 source: EXCEL
 author: Microsoft
 name: Selected range basics
-description: Explores the API for the current selected range
+description: Explores the APIs for the currently selected range
 script:
-  content: |+
-    $("#setup").click(() => invoke(setup));
-
-    $("#get-selection").click(() => invoke(getSelection));
-    $("#set-selection").click(() => invoke(setSelection));
-
-    async function invoke(action: () => Promise<any>) {
-        try {
-            await action();
-        } catch (error) {
-            OfficeHelpers.Utilities.log(error);
-        }
-    }
-
-    async function setup() {
-        await Excel.run(async (context) => {
-            let oldSheet = context.workbook.worksheets.getActiveWorksheet();
-            oldSheet.load("name, position");
-
-            await context.sync();
-
-            let newSheet = context.workbook.worksheets.add();
-            newSheet.position = oldSheet.position;
-            oldSheet.delete();
-            newSheet.name = oldSheet.name;
-            newSheet.activate();
-
-            await context.sync();
-        });
-
-        console.log("Done!");
-    }
+  content: |
+    $("#get-selection").click(getSelection);
+    $("#set-selection").click(setSelection);
 
     async function getSelection() {
-        await Excel.run(async (context) => {
-            let range = context.workbook.getSelectedRange();
-            range.load("address");
+        try {
+            await Excel.run(async (context) => {
+                let range = context.workbook.getSelectedRange();
+                range.load("address");
 
-            await context.sync();
+                await context.sync();
 
-            console.log(`Selected range: ${JSON.stringify(range)}`)
-        });
+                console.log(`Selected range: ${JSON.stringify(range)}`)
```

Comment:
No no no.  Stringify is great for debugging, but if you loaded the `address` property, you know what property you loaded.  So please let's do

`The address of the selected range is "${range.address}"`